l3backend/l3backend.ins

```diff
@@ -110,6 +113,17 @@ and all files in that bundle must be distributed together.
         \from{l3backend-opacity.dtx} {package,luatex}
       }
   }
+\generate
+  {
+    \def\MetaPrefix{--}
+    \usepreamble\luapreamble
+    \usepostamble\luapostamble
+    \file{l3backend-luatex.lua}
+      {
+        \from{l3backend-color.dtx}   {lua,luatex}
+        \from{l3backend-opacity.dtx} {lua,luatex}
+      }
+  }
 \generate
```

Comment:
I'd favour having the Lua part separate from the TeX part, as we do in `l3kernel`.